privacy.md

```diff
@@ -0,0 +1,41 @@
+
+This Privacy Policy describes how we collect, use, and safeguard the information you give us. In this agreement, website refers to the Scratch Addons website (<https://scratchaddons.com/>). Extension refers to our browser extension, which you can download from <https://chrome.google.com/webstore/detail/fbeffbjdlemaoicjdapfpikkikjoneco> or <https://addons.mozilla.org/firefox/addon/scratch-messaging-extension/.>
+
+When we talk about the service, we mean any service provided by us, such as the website or our services used in addons. The terms &quot;we,&quot; &quot;us,&quot; and &quot;our&quot; refer to the leaders of the Scratch Addons open source project (<https://github.com/ScratchAddons>). When you use our services, you agree to this privacy policy.
+
+## Information we collect
+
+We collect a few different kinds of information about you, like what part of the world you are in, what pages you come to our website from, and what device and software you use. We also collect information about how you use our website. We might collect your Scratch username.
+
+### Information collected via Technology
+
+When you load our website, your browser automatically sends us some information about you. That information includes what page you were on before you came to our website, what kind of device you use, and your IP address. This information does not let us know who you are, but it can help us make decisions about how to make our website better. We may use cookies or other local storage to collect information about you, but not who you are, and remember your settings. For example, Scratch Addons may use cookies to remember if you want to use dark mode.
+
+Scratch Addons may use both persistent and session storage. Persistent storage stays on your computer after you close your browser until you delete it, and your browser deletes session storage when you close it. We use Google Analytics to collect data. For information about how Google may use your data, see <https://support.google.com/analytics/answer/6004245> and <https://policies.google.com/technologies/partner-sites>.
+
+### How we use your information
+
+We will never sell, rent, or share your data, except as needed to operate the service. We do share your information with companies whose services we use, like the company that hosts the servers collecting form responses or the one hosting our website. In general, we use the personal information you give to us to help us communicate with you. For example, we might use personal information to contact you in response to questions or provide technical support. If law enforcement asks for data about a user, we will give it to them.
+
+### How we protect information
+
+We are generally quite careful with your data, using security measures like HTTPS and avoiding the use of insecure code that could allow XSS. However, we ensure that hackers won&#39;t. When you use our service, you accept that small have access to your information if we get hacked.
```

Comment:
Does a `&#39;` in an MD file turn into a quote?

Reply:
lol not a serious question - I'll test it now. I just found that interesting.

Reply:
Yes, but I changed it nonetheless.